src/benchmarks/micro/libraries/System.Runtime.Numerics/Perf.BigInteger.cs

```diff
@@ -106,6 +106,44 @@ public IEnumerable<object> ModPowValues()
         public BigInteger ModPow(BigIntegers arguments)
             => BigInteger.ModPow(arguments.Left, arguments.Right, arguments.Other);
 
+        public IEnumerable<object> EqualsValues()
+        {
+            Random rnd = new Random(123456);
+
+            foreach (int byteCount in new[] { 5, 19, 67, 259 })
```

Comment:
It should be enough to benchmark only the larger inputs.

```suggestion
            foreach (int byteCount in new[] { 67, 259 })
```